docs/section4/4-3.md

```diff
@@ -0,0 +1,104 @@
+# ASHVAC – Rule 4-3
+
+**Rule ID:** 4-3
+ 
+**Rule Description:** HVAC Fan Schedules. Schedules for HVAC fans that provide outdoor air for ventilation shall run continuously whenever spaces are occupied.  
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-4 Schedules for the proposed building excluding exception #1. 
+**Appendix G Section Reference:**  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:** 
+
+1. P-RMR hvac systems not modeled to meet the requirements described in Table G3.1 #10 proposed c and d: `if len(applicable_hvac_systems_list_p) > 0`
+2. P-RMR airside HVAC system are been modeled correctly per Table G3.1 #10 proposed c: `if rule 10-10 == "PASS"`
+3. P-RMR airside HVAC system are been modeled correctly per Table G3.1 #10 proposed d: `if rule 10-11 == "PASS"`
+
+**Manual Check:** None  
+**Evaluation Context:** Evaluate each data element  
+**Data Lookup:** None  
+**Function Call:** 
+
+1. match_data_element()
+
+## Rule Logic:
+- For each zone in U_RMR: `for zone_u in U_RMR...zones:`
+    - Reset heating system exists boolean variable equal to false each time it loops to a different zone: `heating_system_exists_u = FALSE`
+    - Reset cooling system exists boolean variable equal to false each time it loops to a different zone: `cooling_system_exists_u = FALSE`
+    - Reset heating system exists boolean variable equal to false each time it loops to a different zone: `heating_system_exists_p = FALSE`
+    - Reset cooling system exists boolean variable equal to false each time it loops to a different zone: `cooling_system_exists_p = FALSE`
+    - Check if there are terminal units associated with the zone, if not skip to proposed: `if len(zone_u.terminals) != 0:`
+        - For each terminal unit associated with each zone: `for terminal_u in zone_u:`
+            - if reheat_source is not null then set heating exists boolean variable equal to true: `if terminal_u.reheat_source != "None":`
+                - Set heating system exists boolean variable equal to true: `heating_system_exists_u = TRUE`
+            - if heat_capacity is not zero then set heating exists boolean variable equal to true: `if terminal_u.heat_capacity != 0:`
```

Comment:
i'm assuming here we're checking if a heating ssystem exisits or has been designed. 2 questions/suggestions
1. Change the term heating_system_exists_u to heating_system_exists_is-designed_u . the term exists implies that we're system is modeled as existing. If thats what the intent is, then please ignore.
2. we're checking both the reheat source and the heat capacity, is that to account for both reheat systems and single zone systems?

Reply:
@claperle since the logic is so convoluted and has so many steps, it might help to have a statement right under "Rule Logic" that states "Determine if a HVAC system has been designed or modeled as exisiting in U-RMR" and then right before line 50
. add a statement that says "Determine if the same  zone has been analyzed with a HVAC system in P-RMR" or something similar to clarify the intent.

Reply:
All of these have been addressed. Good suggestions! Thanks.